source/2020-05-08-the-ember-times-issue-147.md

```diff
@@ -64,14 +65,14 @@ In case you missed, the RFCs for [Embroider](https://github.com/emberjs/rfcs/pul
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [ember-keyboard RFC ⌨️](https://github.com/adopted-ember-addons/ember-keyboard/pull/135)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+If you've ever wondered about some of the finer details of the `code` and `key` properties of keyboard events in modern browsers you may want to check out [Luke Melia's (@lukemelia)](https://github.com/lukemelia) recent [RFC](https://github.com/adopted-ember-addons/ember-keyboard/pull/135) for `ember-keyboard`. 
```

Comment:
Because the sentence is long, I think adding a comma may help some of the readers.
```suggestion
If you've ever wondered about some of the finer details of the `code` and `key` properties of keyboard events in modern browsers, you may want to check out [Luke Melia's (@lukemelia)](https://github.com/lukemelia) recent [RFC](https://github.com/adopted-ember-addons/ember-keyboard/pull/135) for `ember-keyboard`. 
```